src/argparse.nim

```diff
@@ -232,6 +232,8 @@ proc genHelp(builder: Builder):string {.compileTime.} =
     result.add(opts)
     result.add("\L")
 
+  result.setLen(result.high)
```

Comment:
Would `result = result.strip()` make more sense here?  I had to do a bunch of digging to figure out what this did.  Though `.setLen` operates in place, this happens at compile time so it may not be a big deal to reassign result.

Reply:
turns out `strutils` already has [stripLineEnd](https://nim-lang.github.io/Nim/strutils.html#stripLineEnd%2Cstring) which does similar thing